docs/how-to-guides/azure_resource_provision.json

```diff
@@ -349,6 +353,48 @@
                 "principalId": "[parameters('principalId')]",
                 "scope": "[resourceGroup().id]"
             }
+        },
+        {
+            "type": "Microsoft.Web/serverfarms",
+            "apiVersion": "[variables('webAppAPIVersion')]",
+            "name": "[variables('webAppPlanName')]",
+            "location": "[variables('location')]",
+            "sku": {
+                "name": "[variables('webAppPlanSku')]"
+            },
+            "kind": "linux",
+            "properties": {
+                "reserved": true
+            }
+        },
+        {
+            "type": "Microsoft.Web/sites",
+            "apiVersion": "[variables('webAppAPIVersion')]",
+            "name": "[variables('webAppName')]",
+            "location": "[variables('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Web/serverfarms', variables('webAppPlanName'))]"
+            ],
+            "kind": "app,linux,container",
+            "identity": {
+                "type": "systemAssigned"
+            },
+            "properties": {
+                "name": "[variables('webAppName')]",
+                "httpsOnly": "true",
+                "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('webAppPlanName'))]",
+                "siteConfig": {
+                    "linuxFxVersion": "DOCKER|dockercloud/hello-world",
```

Comment:
For integration let's use this docker image name blrchen/feathr-sql-registry

Should move this to parameter so it can be overrided. As currently sql registry and purview registry are different images. Furthermore, in the further, we might use a different name spaces under org account.

Reply:
Fixed